go.sum

```diff
@@ -147,6 +147,8 @@ github.com/cockroachdb/logtags v0.0.0-20190617123548-eb05cc24525f h1:o/kfcElHqOi
 github.com/cockroachdb/logtags v0.0.0-20190617123548-eb05cc24525f/go.mod h1:i/u985jwjWRlyHXQbwatDASoW0RMlZ/3i9yJHE2xLkI=
 github.com/cockroachdb/pebble v0.0.0-20200624214307-560ed0b8a18c h1:MQVYGjMEPj6TRGTi05ct75tDhbBKKFoJ0yKvvEZzIZw=
 github.com/cockroachdb/pebble v0.0.0-20200624214307-560ed0b8a18c/go.mod h1:crLnbSFbwAcQNs9FPfI1avHb5BqVgqZcr4r+IzpJ5FM=
```

Comment:
missing a `go mod tidy`

Reply:
(instruction for that included if you rebase i think!)

Reply:
Done.